erudit/templates/userspace/editor/form.html

```diff
@@ -8,6 +8,10 @@
 {% endblock %}
 
 {% block extrahead %}
+<script src="{% static 'vendor/jquery/dist/jquery.min.js' %}"></script>
```

Comment:
I already mentioned it to @cormier to not include the last commit while merging back this PR.